src/webgpu/api/operation/copyBetweenLinearDataAndTexture.spec.ts

```diff
@@ -1013,69 +1014,43 @@ g.test('copy_various_mip_levels')
 // slice we can set rowsPerImage to 0. Origin, offset, mipLevel and rowsPerImage values will be set
 // to undefined when appropriate.
 g.test('copy_with_no_image_or_slice_padding_and_undefined_values')
+  .desc(
+    `TODO: description
+
+  TODO: "0" below really means "undefined" (it gets replaced with undefined later).
```

Comment:
Didn't you just do this cleanup?

Reply:
Only partly, I only did it for bytesPerRow/rowsPerImage (in the params, but not in the rest of the test code). Though I suppose I can easily do it for the others too right now.